README.md

```diff
@@ -112,3 +112,20 @@ spec:
 [tilt]: https://tilt.dev
 [capi_tilt]: https://cluster-api.sigs.k8s.io/developer/tilt.html
 [k8s_quantity_docs]: https://kubernetes.io/docs/reference/kubernetes-api/common-definitions/quantity/
+
+## Running Integration (E2E) Test
+
+```bash
+export ROOT_DIR=<path to cluster-api-provider-kubemark repository>
+cd $(ROOT_DIR)
+make test-e2e 
```

Comment:
should this be `make -C hack/tests` ?

Reply:
Updated $(ROOT_DIR)/Makefile. Please check.

Reply:
I think just `make test-e2e` since main Makefile also contains target `test-e2e`

Reply:
thanks for the updates